recipe/meta.yaml

```diff
@@ -39,12 +39,14 @@ test:
     - IF NOT EXIST %LIBRARY_BIN%\sqlite3.dll exit 1  # [win]
     - IF NOT EXIST %LIBRARY_LIB%\sqlite3.lib exit 1  # [win]
     - IF NOT EXIST %LIBRARY_INC%\sqlite3.h exit 1  # [win]
+    #- conda inspect linkages -p $PREFIX sqlite  # [not win]
+    #- conda inspect objects -p $PREFIX sqlite  # [osx]
```

Comment:
I have to be honest. `conda inspect` has left a bad taste in my mouth. Mainly because it was broken for some time ( https://github.com/conda/conda-build/issues/836 ) then was "fixed" though we don't know how and remain unclear on what guarantees we actually have for using it. Now we also have this pain. ( https://github.com/conda/conda-build/issues/1248 ) Admittedly the latter should be fixed by a `conda-build` upgrade.

Would rather see this in a separate PR where it can be discussed not snuck in with a version bump.